tests/constants/dockercli/cmd.go

```diff
@@ -71,8 +71,11 @@ const (
 	// RemoveContainer remove the container
 	RemoveContainer = docker + "rm -f "
 
+	// ContainerImage busybox container image
+	ContainerImage = " busybox "
+
 	// TestContainer test busybox container that keeps running
-	TestContainer = " busybox tail -f /dev/null "
+	TestContainer = ContainerImage + " tail -f /dev/null "
```

Comment:
Yes we can have a common teardown function. And if need be to have another container with given sleep time, we can add as required. Keeping this as it is but moving to separate PR #1320